docs/DeveloperGuide.md

```diff
@@ -605,6 +605,43 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Incrementing salaries of multiple employees
+
+1. Incrementing salaries of all employees in the displayed list of employees.
+
+   1. Test case: `increment 1000`<br>
+      Prerequisite: Salaries of all employees do not exceed the maximum salary after increasing by 1000. <br>
+      Expected: Salaries of all employees in the list increased by 1000.
+   
+   1. Test case: `increment -10000` <br>
+      Prerequisite: Salary of at least one employee in the displayed list is below 10000. <br>
+      Expected: No change in salaries of all employees. Error details shown in the status message.
+
+### Undoing a modification
+
+1. Undo the previous command that caused a modification in employee data.
+
+   1. Test case: `undo`<br>
```

Comment:
Shall we have a two step test case? A command that modifies the list followed by undo, something like 
Test case: `delete 1` followed by `undo`
Expected: The list is reverted to before delete 1 was executed
This makes the "Expected" more controlled so users know what to expect. 

In addition, maybe you can have a one line explanation on what commands cause a modification, something like this
Commands that cause a modification in employee data: `add`, `delete`, `edit`....

Reply:
Thanks for the suggestion! I added the multiple step test case for both `undo` and `redo`!

Reply:
I think there isn't a need to include the commands the cause a modification as this was already mentioned in the user guide and this section is meant to be used with the user guide

Reply:
Yepp okay that makes sense, LGTM